vscode-extension/src/extension.ts

```diff
@@ -757,18 +757,29 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
     return;
   }
 
+  const envTemplatePath = vscode.Uri.joinPath(
+    context.extensionUri,
+    "static",
+    "env_template.env"
+  );
+
   if (fs.existsSync(envPath)) {
-    vscode.window.showInformationMessage(
-      "Env file already exists, will implement next PR"
-    );
+    const helperText = (
+      await vscode.workspace.fs.readFile(envTemplatePath)
+    ).toString();
+
+    // TODO: Check if we already appended the template text to existing .env
+    // file before. If we did, don't do it again
+    fs.appendFile(envPath, "\n\n" + helperText, function (err) {
```

Comment:
This is fine for now, but if a file already exists perhaps we should ask the user to specify a different file or ask them if they want to overwrite or append. Can be a future improvement.

Reply:
I'm going to push back saying this is P1 and focus more on content for tonight. If I have time before midnight I'll do it